src/libs/actions/PaymentMethods.js

```diff
@@ -130,7 +130,7 @@ function transferWalletBalance(paymentMethod) {
     };
     Onyx.merge(ONYXKEYS.WALLET_TRANSFER, {loading: true});
 
-    API.TransferWalletBalance(parameters)
+    API.Wallet_TransferBalance(parameters)
```

Comment:
Instead of changing this can you please fix the export name so it matches the command?

The command name is`TransferWalletBalance` so the `API` method should be `API.TransferWalletBalance()`

Reply:
I thought of following the existing code. But I can do that.

Reply:
Thanks.